lib/game_logic.rb

```diff
@@ -0,0 +1,52 @@
+class TicTacToe
```

Comment:
This is a very nice, concise and readable code, but please split it into multiple classes. While doing that please take into account:

1. The code should respect the single responsibility rule ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/single_responsibility.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).
2.  Be aware of using private methods and attr_readers/attr_writers instead of attr_accessors if needed ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone3/attr_accessor.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59565018/)).